src/workerd/api/streams/compression.c++

```diff
@@ -397,11 +398,14 @@ jsg::Ref<DecompressionStream> DecompressionStream::constructor(
     kj::String format) {
   JSG_REQUIRE(format == "deflate" || format == "gzip" || format == "deflate-raw", TypeError,
                "The compression format must be either 'deflate', 'deflate-raw' or 'gzip'.");
+  auto& ioContext = IoContext::current();
+  auto flags = ioContext.getWorker().getIsolate().getApiIsolate().getFeatureFlags();
```

Comment:
I'm somewhat suspicious of this, mainly because I haven't seen feature flags be accessed like this before. I suppose if it works it's fine, but iirc jsg will get you the feature flags if you just put them in the method (`CompatibilityFlags::Reader featureFlags`) and I see a lot of other code doing it this way so might be better to be consistent.